testing/regress/ecl/redislockingtest.ecl

```diff
@@ -111,7 +111,7 @@ SEQUENTIAL(
 //Test unlock
 SEQUENTIAL(
     myRedis.FlushDB(),
-    myRedis.GetOrLockString('testlock'),/*by default lock expires after 1s*/
+    myRedis.GetOrLockString('testlock',, 1000),
```

Comment:
This won't yield the same result as that in the key until the 5.2.8 (7701) change is upmerged.